### PR TITLE
LPAL-421 feedback api

### PR DESCRIPTION
**Pull request by nickdavis2001**

### CODE REVIEW

File: .github/workflows/build_images.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  security-events: none
+  statuses: none
+
+jobs:
+  build-images:
+    name: Build docker images
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
```
Comment (williamfalconeruk) [inline .github/workflows/build_images.yml]: try `contents: write` here job permissions are overwriting top level perms
```suggestion
      contents: write
```

File: .github/workflows/build_images.yml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+          RELEASE_BRANCHES: main
+          WITH_V: true
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID_ACTIONS }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY_ACTIONS }}
+          aws-region: eu-west-1
+          role-to-assume: arn:aws:iam::311462405659:role/sirius-actions-ci
```
Comment (williamfalconeruk) [inline .github/workflows/build_images.yml]:
```suggestion
          role-to-assume: arn:aws:iam::311462405659:role/ci
```

File: .github/workflows/build_images.yml [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+          RELEASE_BRANCHES: main
+          WITH_V: true
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID_ACTIONS }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY_ACTIONS }}
+          aws-region: eu-west-1
+          role-to-assume: arn:aws:iam::311462405659:role/ci
```
Comment (williamfalconeruk) [inline .github/workflows/build_images.yml]:
```suggestion
          role-to-assume: arn:aws:iam::311462405659:role/flask-actions-ci
```

**Comment by nickdavis2001** [comment]
Work now proceeding on different branch

**Pull request closed without merging**